src/dbnode/topology/testutil/topology.go

```diff
@@ -28,6 +28,11 @@ import (
 	"github.com/m3db/m3cluster/shard"
 )
 
+const (
+	// SelfID is the string used to represent the ID of the origin node.
+	SelfID = "self"
```

Comment:
Is there a reason why we created this concept, instead of just pass around the original topology map and the identifier of the local node?

Reply:
Oh this is just a test util package, please ignore my comment.